perf/benchmark/setup_test.sh

```diff
@@ -55,12 +56,13 @@ function setup_test() {
       --set rbac.enabled="${RBAC_ENABLED}" \
       --set namespace="${NAMESPACE}" \
       --set loadGenType="${LOAD_GEN_TYPE}" \
-      --set excludeOutboundIPRanges=$(pod_ip_range)\
-      --set includeOutboundIPRanges=$(svc_ip_range) \
```

Comment:
Had trouble on cncf cluster with services being assigned to IPs outside this range

Reply:
can you split this into another PR?

Reply:
I gotta rebase and change the title. The rustfilt stuff is already merged